docs/source/contributing/pr_tutorial.md

```diff
@@ -26,6 +26,15 @@ The preferred workflow for contributing to PyMC is to fork the [GitHub repositor
 
 4. Project requirements are in ``requirements.txt``, and libraries used for development are in ``requirements-dev.txt``. The easiest (and recommended) way to set up a development environment is via [miniconda](https://docs.conda.io/en/latest/miniconda.html):
 
+   If using Windows:
+   ```bash
+   conda env create -f .\conda-envs\windows-environment-dev-py38.yml
+   conda activate pymc-dev-py38
+   pip install .
```

Comment:
i think this needs to be `pip install -e` for contriibutions otherwise itll install the package **as is** and any further modifications will not be reflected

https://stackoverflow.com/questions/42609943/what-is-the-use-case-for-pip-install-e

Reply:
updated pr_tutorial.md with suggested "pip install -e ." line

Reply:
Thank you